### ISSUE TITLE
[BUG] error: bundling: NotFoundError: Cannot find entry file index.ios.js in any of the roots

### ISSUE THREAD

**Issue by mrgoonie** [facebook/metro-bundler#5]
When I run the react native app on the simulator by `react-native run-ios`, this error appear:

![image](https://user-images.githubusercontent.com/6857382/26960519-38a8f3fc-4d02-11e7-85c8-7746b1a5886e.png)

```
error: bundling: NotFoundError: Cannot find entry file index.ios.js in any of the roots: ["/Users/goon/HelloRN/node_modules/react-native/packager"]
    at DependencyGraph._getAbsolutePath (/Users/goon/HelloRN/node_modules/react-native/packager/src/node-haste/DependencyGraph.js:280:11)
    at DependencyGraph.getDependencies (/Users/goon/HelloRN/node_modules/react-native/packager/src/node-haste/DependencyGraph.js:218:26)
    at Resolver.getDependencies (/Users/goon/HelloRN/node_modules/react-native/packager/src/Resolver/index.js:107:27)
    at /Users/goon/HelloRN/node_modules/react-native/packager/src/Bundler/index.js:591:37
    at next (native)
    at step (/Users/goon/HelloRN/node_modules/react-native/packager/src/Bundler/index.js:12:445)
    at /Users/goon/HelloRN/node_modules/react-native/packager/src/Bundler/index.js:12:605
    at process._tickCallback (internal/process/next_tick.js:103:7)
Bundling `index.ios.js`  0.0% (0/1), failed.
```

What I've tried:
```
watchman watch-del-all
rm -rf node_modules && npm install
npm start -- --reset-cache
```
then
`react-native run-ios`

..even tried to re-install react & react native

But it's still not working.

**Comment by cpojer** [comment]
Please note this issue tracker is not a help forum. We recommend using StackOverflow for questions. Thank you :)

**Comment by switchpt** [comment]
@cpojer Looks like an issue to me.  react-native run-ios doesn't work after upgrade. Please see issue https://github.com/facebook/react-native/issues/14314 as well.

**Comment by mrgoonie** [comment]
That's funny when @skevy told us to open the issue over here (ref: https://github.com/facebook/react-native/issues/14314), but then @cpojer close my issue and told me to open it... somewhere else??

**Comment by switchpt** [comment]
@mrgoonie  seems to be working fine with react-native 0.45.1 version

**Comment by joxad** [comment]
Got the same problem after update from react-native 16.0.0-alpha.6 to 16.0.0-alpha.12

**Comment by grabbou** [comment]
It was a regression that was fixed in `0.45.1`. For reference, here's the commit that fixes it: https://github.com/facebook/react-native/commit/39e0750223a46ba39f7f22f5d8a49f5aca33e378

**Comment by baba43** [comment]
I have the same error with react-native 0.46.1 and also 0.47.1 on Windows (created with create-react-native-app).
